### PR TITLE
Added necessary code for postgres-12 upgrade

### PR DESCRIPTION
**Pull request by JoshingYou1**

### CODE REVIEW

File: ruby/2.7.5/Dockerfile [hunk 1]
```diff
@@ -41,9 +42,13 @@ RUN yum install -y \
         redis \
         sqlite-devel \
         readline-devel
-
-RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-RUN yum install -y postgresql-server12 postgresql-contrib12 postgresql-devel12
+COPY VA-Internal-S2-RCA1-v1.cer /etc/pki/ca-trust/source/anchors/VA-Internal-S2-RCA1-v1.cer
```
Comment (aurora-a-k-a-lightning) [inline ruby/2.7.5/Dockerfile:47]: i think a note should be added here that this assumes this Dockerfile is being ran in the VAEC...

Comment (aurora-a-k-a-lightning) [reply ruby/2.7.5/Dockerfile:47]: from lines 45 - 47 should be noted

Comment (JoshingYou1) [reply ruby/2.7.5/Dockerfile:47]: Okay, I added a comment. Thanks Aurora! 😄

Comment (aurora-a-k-a-lightning) [reply ruby/2.7.5/Dockerfile:47]: excellent! LGTM!!!

**Review by aurora-a-k-a-lightning — APPROVED** [review]
LGTM

**Review by bradjohnson92008 — APPROVED** [review]
Thanks for doing this Josh - LGTM!

**Review by camillevilla — APPROVED** [review]

**Pull request merged**